mod.ts

```diff
@@ -41,6 +41,9 @@
  * #### Worker
  *
  * ```typescript
+ * /// <reference no-default-lib="true" />
+ * /// <reference lib="deno.worker" />
```

Comment:
Do we need this? For what?

Reply:
By giving this instruction to scripts that assume a Worker environment, global objects and the like will be typed correctly. If not present, `self` is the DOM Window.